glide.yaml

```diff
@@ -19,7 +17,7 @@ import:
 - package: github.com/tendermint/go-merkle
   version: develop
 - package: github.com/tendermint/go-p2p
-  version: develop
+  version: viper
```

Comment:
... yes I would actually appreciate some advice as to how to deal with managing multi-repo pull requests, obviously as soon as `go-p2p` has `viper` merged into `develop` then this should be changed back, but until then it makes sense to be left on `viper` while in PR - Is there a better way of doing this?